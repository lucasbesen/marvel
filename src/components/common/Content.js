import { withStyles } from '@material-ui/core';
import Paper from '@material-ui/core/Paper';

const Content = withStyles({
  root: {
    width: '720px',
    maxWidth: '720px',
    height: '500px',
    maxHeight: '500px',
    overflowY: 'auto',
  },
})(Paper);

export default Content;
