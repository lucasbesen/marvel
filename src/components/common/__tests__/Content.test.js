import React from 'react';
import renderer from 'react-test-renderer';

import Content from '../Content';

it('should render content without error', () => {
  const component = renderer.create(
    <Content>
      <p>Testing Content component</p>
    </Content>,
  );

  const tree = component.toJSON();

  expect(tree).toMatchSnapshot();
});
