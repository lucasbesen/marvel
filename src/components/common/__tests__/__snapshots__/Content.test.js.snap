// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render content without error 1`] = `
<div
  className="MuiPaper-root WithStyles(ForwardRef(Paper))-root-1 MuiPaper-elevation1 MuiPaper-rounded"
>
  <p>
    Testing Content component
  </p>
</div>
`;
